backend/src/main/kotlin/app/ehrenamtskarte/backend/verification/service/CardVerifier.kt

```diff
@@ -19,7 +19,7 @@ object CardVerifier {
     }
 
     private fun verifyCard(card: CardEntity, totp: Int): Boolean =
```

Comment:
Just for convenience: You can omit the return type in cases where the right side is an expression